teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorOptions.java

```diff
@@ -30,10 +33,18 @@
       converter = GraffitiConverter.class,
       paramLabel = "<GRAFFITI STRING>",
       description =
-          "Graffiti to include during block creation (gets converted to bytes and padded to Bytes32).",
+          "Graffiti value to include during block creation. Value gets converted to bytes and padded to Bytes32).",
       arity = "1")
   private Bytes32 graffiti;
 
+  @Option(
+      names = {"--validators-graffiti-file"},
+      paramLabel = "<GRAFFITI FILE>",
+      description =
+          "File to load graffiti value to include during block creation. Value gets converted to bytes and padded to Bytes32).  If file reading fails during block creation, teku will fall back to any value supplied via --graffiti.",
```

Comment:
Another stray parenthesis:
```suggestion
          "File to load graffiti value to include during block creation. Value gets converted to bytes and padded to Bytes32.  If file reading fails during block creation, teku will fall back to any value supplied via --graffiti.",
```